follow_repos_by_search_term.py

```diff
@@ -0,0 +1,67 @@
+from typing import List
+from github import Github
+from lgtm import LGTMSite
+from datetime import datetime
+
+import sys
+import yaml
+import time
+
+def create_github() -> Github:
+    with open("config.yml") as config_file:
+        config = yaml.safe_load(config_file)
+        github: dict = config['github']
+        return Github(github['api_key'])
```

Comment:
Can we extract this to a common script?

Reply:
Done! I also extracted the date logic into a separate script and created a `utils` folder to store utility-related scripts.